include/eos/morphablemodel/coefficients.hpp

```diff
@@ -50,6 +50,26 @@ inline void save_coefficients(std::vector<float> coefficients, std::string filen
     output_archive(cereal::make_nvp("shape_coefficients", coefficients));
 };
 
+/**
+ * Loads coefficients (for example PCA shape coefficients) from a json file.
+ *
+ * @param[in] filename The file to write.
+ * @throws std::runtime_error if unable to open the given file for reading.
+ * @return Returns vector of floats.
+ */
+inline std::vector<float> load_coefficients(std::string filename)
+{
+    std::vector<float> coefficients;
+    std::ifstream file(filename);
+    if (file.fail())
```

Comment:
I see you copied this from the `save_` method. I can't believe there was still a check in eos that used `.fail()`! I changed all of them but must've missed this one. Please change this to just `if (file)`, and if you could change the `save_` method too!